addon/components/table-column.js

```diff
@@ -77,53 +85,31 @@ export default Component.extend({
     this._super(...arguments);
     assert('Must use table column as a child of table-columns or fixed-table-columns.', this.parentView);
     run.scheduleOnce('actions', this, this._registerWithParent);
-
-    this._setValueDependentKeys();
   },
 
   /**
-    Sets dependent keys on the _value computed property. This is done since
-    we need to clear the cached version based on a dynamic key, and volatile()
-    does not do the needful.
-    @private
-  **/
-  _setValueDependentKeys() {
-    const path = this.get('valueBindingPath');
-    const pathKey = `row.${path}`;
-    this._value.property('valueBindingPath', 'row', pathKey);
-  },
-
-  /**
-    Return the value for the cell based on row.valueBindingPath. Only used if
-    a block is not passed, the path is provided, and the row is not empty.
-    @private
-  */
-  _value: computed('valueBindingPath', 'row', function() {
-    const path = this.get('valueBindingPath');
-    const row = this.get('row');
-    const hasBlockParams = this.get('hasBlockParams');
-
-    if (hasBlockParams || isEmpty(path) || isEmpty(row)) {
-      return null;
-    }
-
-    return get(row, path);
-  }),
-
-  /**
-    Register this column with its parent view.
+    Register this column with its parent table.
     @private
   */
   _registerWithParent() {
-    let parent = this.get('parentView');
-    parent.registerColumn(this);
-    this.set('_registeredParent', parent);
+    // temp hack since we're no longer a direct child
+    let table = this.get('parentView.parentView.parentView');
```

Comment:
This should be improved on later (forcing 2.3 and contextual components is likely the best future option), but near term we should find a way to do this that doesn't use private properties.